src/platform.js

```diff
@@ -35,17 +36,28 @@ var utils = require('./utils');
 *
 * @module enyo/platform
 */
-exports = module.exports =
-	/** @lends module:enyo/platform~platform */ {
-	//* `true` if the platform has native single-finger [events]{@glossary event}.
+exports = module.exports = {
+	/**
+	* `true` if the platform has native single-finger [events]{@glossary event}.
+	* @public 
+	*/
 	touch: Boolean(('ontouchstart' in window) || window.navigator.msMaxTouchPoints),
-	//* `true` if the platform has native double-finger [events]{@glossary event}.
+	/**
+	* `true` if the platform has native double-finger [events]{@glossary event}.
+	* @public
+	*/
 	gesture: Boolean(('ongesturestart' in window) || window.navigator.msMaxTouchPoints)
+
+	/**
+	* The name of the platform that was detected or `undefined` if the platform
+	* was unrecognized. This value is the key name for the major version of the
+	* platform on the exported object.
+	* @var {string} platformName
```

Comment:
Gotcha - for some reason that synonym didn't come up in the search. I'm not opposed to it, but we seem to be using `@member` everywhere else.